src/porepy/numerics/contact_mechanics/contact_conditions.py

```diff
@@ -84,22 +113,31 @@ def discretize(self, g_h, g_l, data_h, data_l, data_edge):
         #   gut feel says yes, but I'm not sure.
 
         # Process input
-        parameters_l = data_l[pp.PARAMETERS]
-        friction_coefficient = parameters_l[self.keyword]["friction_coefficient"]
-
-        cohesion = parameters_l[self.keyword].get("cohesion", 0)
+        parameters_l = data_l[pp.PARAMETERS][self.keyword]
 
+        # Mandatory cellwise friction coefficient relates normal and tangential forces.
```

Comment:
If the friction coefficient is 'mandatory cellwise' - should we then delete the array expansion right below?

Reply:
I think it's ok to expand from constant value for each fracture, since this will be the most common case.